ui/src/app/utils/access-utils.tsx

```diff
@@ -90,62 +90,62 @@ interface AccessModuleConfig {
 // without a next step:
 // https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java#L240-L272
 export const getAccessModuleConfig = (moduleName: AccessModule): AccessModuleConfig => {
-  const {enableRasLoginGovLinking, enableEraCommons, enableComplianceTraining} = serverConfigStore.get().config;
```

Comment:
I think the indent size should be 2 per: 
https://github.com/all-of-us/workbench/blob/master/ui/intellij-typescript-style.xml
But let me know that I should revert this.

Reply:
I don't think we have a linter reliably enforcing this right now.  Until we fix that, I'm OK with updating spacing to fit your local environment.

Reply:
I got that from my intellij: https://docs.google.com/document/d/1DtESBapEzvuti7xODTFPHorwmLM7LybF-6D5lhbIkLU/edit#

Reply:
I think that doc is part of onboarding process when setting up local development

Reply:
Thanks!  I thought I had that configured but I must have lost it somehow.